spec/models/user_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) { FactoryGirl.create(:user) }
+  let(:user_2) { FactoryGirl.create(:user) }
+
+  it 'is invalid without a name' do
+    user_1.name = nil
+    expect(user_1.valid?).to be(false)
+    expect(user_1.errors[:name]).to include("can't be blank")
+  end
+
+  it 'is invalid without an email address' do
+    user_1.email = nil
+    expect(user_1.valid?).to be(false)
+    expect(user_1.errors[:email]).to include("can't be blank")
+  end
+
+  it 'is invalid with a duplicate email address' do
+    user_1.save
```

Comment:
it is already saved. why do you save it again?

Reply:
on next commit will be fixed